pkgs/development/python-modules/autobahn/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildPythonPackage, fetchurl, isPy3k,
+  unittest2, mock, pytest, trollius, pytest-asyncio,
+  six, twisted, txaio
+}:
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "autobahn";
+  version = "0.18.2";
+
+  src = fetchurl {
+    url = "mirror://pypi/a/${pname}/${name}.tar.gz";
+    sha256 = "1alp71plqnrak5nm2vn9mmkxayjb081c1kihqwf60wdpvv0w7y14";
+  };
+
+  buildInputs = [ unittest2 mock pytest trollius pytest-asyncio ];
+  propagatedBuildInputs = [ six twisted txaio ];
+
+  disabled = !isPy3k;
```

Comment:
Yes, but tests will run `pytest-asyncio`, which contains code with python2 incompatible syntax (and incompatible language anyway). This is different from runtime dependencies needed to run `autobahn`. It might well work with python2 install but we cannot check it with the provided test suite.

I am updating to the recent release of autobahn and fix runtime dependencies.